checkov/terraform/checks/resource/azure/AppGWUseWAFMode.py

```diff
@@ -19,5 +21,8 @@ def scan_resource_conf(self, conf):
                 return CheckResult.FAILED
         return CheckResult.PASSED
 
+    def get_evaluated_keys(self) -> List[str]:
```

Comment:
I think you forgot to remove the `evaluated_keys` assignment in `scan_resource_conf` function